docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/local-registry.md

```diff
@@ -78,6 +80,89 @@ to <VersionedLink text="Harbor Edge-Native Config pack" url="/integrations/packs
 7. In the `harbor-config.storage` parameter, make sure you allocate enough storage in the `registry` field to store all
    your images.
 
+### Provide Credentials to Private Registries (Optional)
+
+If your cluster profile does not reference a private registry for images or helm charts or you have configured the
+Palette agent to download all images from an external private registry, you can skip this step.
+
+:::info
+
+Configuring external registries through user data will instruct the Palette agent to download all images from the
+external registry and prepend the registry URL to every image referenced in the cluster profile. Therefore, if you have
+configured an external registry through user data, you no longer need to provide credentials to the private registries
+referenced in the profile because those private registries will not be used. You are instead responsible for ensuring
+that all images are present in the external registry. For more information, refer to
+[Deploy with External Private Registry](./deploy-external-registry.md).
+
+:::
+
+If your have not configured an external registry for your Edge host through the Edge installer configuration user data,
+and your cluster profile references private registries, you must provide credentials to those private registries through
+Kubernetes secrets using a manifest in your cluster profile.
```

Comment:
```suggestion
[Kubernetes secrets](https://kubernetes.io/docs/concepts/configuration/secret/) using a manifest in your cluster profile.
```
I think adding a link to the k8s secret ref page might help those that may not understand what a k8s secret is 😄